internal/app/connectconformance/testsuites/data/grpc_web_client_trailers.yaml

```diff
@@ -143,3 +145,139 @@ testCases:
       responseTrailers:
         - name: x-custom-trailer
           value: [ "bing", "quuz" ]
+
+  # Server-streaming: Trailers and status are in body (no other response messages)
+  - request:
+      testName: server-stream/trailers-in-body/expected
```

Comment:
For increased clarity of test case names (and more consistency/uniformity), could you rename the existing ones to have a `"unary/"` prefix?

Reply:
Are we okay with potentially breaking known-failing lists?

Reply:
Yes.
https://github.com/connectrpc/conformance?tab=readme-ov-file#status-stable

Reply:
Excellent. Done in https://github.com/connectrpc/conformance/pull/935/commits/911c055d30ee47dbb2800ede795057e981b7aa25.